app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -185,7 +185,16 @@
                   canRemoveItem=false
                 }}
               {{else}}
-                {{input type=field.type value=(mut (get data.speaker field.fieldIdentifier)) id=(if field.isRequired (concat 'speaker_' field.fieldIdentifier '_required') (concat 'speaker_' field.fieldIdentifier))}}
+                {{#if (eq field.name 'Twitter')}}
+                  {{widgets/forms/link-input
+                    fixedName=true
+                    inputId=(if field.isRequired (concat 'speaker_' field.fieldIdentifier '_required') (concat 'speaker_' field.fieldIdentifier))
+                    segmentedLink=data.speaker.segmentedLinkTwitter
```

Comment:
@shreyanshdwivedi This line will be affected by the following change If i reduce the if condition according to your suggestion

Reply:
@kushthedude I think you can use an `if` condition here?

Reply:
The following `if` I used before?

Reply:
Something similar to line 191. Use it with the if condition I commented earlier. This way you can merge both the code blocks into one and remove code duplication